hsa-ext-rocr/PKGBUILD

```diff
@@ -3,16 +3,16 @@
 # Contributor: Ranieri Althoff <ranisalt+aur at gmail.com>
 
 pkgname=hsa-ext-rocr
-pkgver=3.5.1
+pkgver=3.5.0
```

Comment:
The AUR will not accept an update to the PKGBUILD where the pkgver has decreased. I suggest for now we keep the pkgver at 3.5.1 and bump the pkgrel by 1 even as we change to the working url. While technically incorrect, AMD will release a new version soon enough.

Reply:
Reasonable, done! @acxz